schemas/2014-04-01-preview/deploymentTemplate.json

```diff
@@ -40,6 +40,8 @@
                             },
                             {
                                 "oneOf": [
+                                    { "$ref": "http://schema.management.azure.com/schemas/2014-05-21-preview/Microsoft.DevTestLabs.json#/definitions/labs" },
```

Comment:
The schema file added is 2015-05-21-preview/Microsoft.DevTestLabs.json, so the reference should be "http://schema.management.azure.com/schemas/2015-05-21-preview/Microsoft.DevTestLabs.json#/definitions/labs", instead of "2014-05-21-preview"